SourceX/main.cpp

```diff
@@ -29,6 +29,14 @@ static std::string build_cmdline(int argc, char **argv)
 	return str;
 }
 
+#if defined(__3DS__)

```

Comment:
@MrHuu https://github.com/MrHuu/devilutionX-3ds/pull/2

A simpler fix there - using a lambda

Reply:
Auwch.. didn't notice your fix before commit. I'll have it cleaned up.

Reply:
No worries, I'm rebasing my PR

Reply:
I think you didn't notice because you're not "Watching" your fork for some reason

Reply:
Rebased